src/components/Swap/Input.tsx

```diff
@@ -135,11 +133,11 @@ export default function Input() {
       >
         <ThemedText.Body2 color="secondary" userSelect>
           <Row>
-            <USDC isLoading={isRouteLoading}>{usdc ? `$${formatCurrencyAmount(usdc, 6, 'en', 2)}` : ''}</USDC>
+            <USDC isLoading={isRouteLoading}>{usdc ? `${formatCurrencyAmount(usdc, true)}` : ''}</USDC>
```

Comment:
Can this be `used &&`? I don't think it will render the falsey value, but you'd need to test it.

Reply:
yes